set.mm

```diff
@@ -30137,8 +30137,16 @@ choice between (what we call) a "definitional form" where the shorter
     vtocl2g.2 $e |- ( y = B -> ( ps <-> ch ) ) $.
     vtocl2g.3 $e |- ph $.
     $( Implicit substitution of 2 classes for 2 setvar variables.  (Contributed
-       by NM, 25-Apr-1995.) $)
+       by NM, 25-Apr-1995.)  Remove dependency on ~ ax-10 , ~ ax-11 , and
+       ~ ax-13 .  (Revised by Steven Nguyen, 29-Nov-2022.) $)
```

Comment:
Thanks for doing that. I was not very careful when I removed these dependencies from vtoclg. There are certainly similar cases that could benefit from it. Anyway, I should move bj-elissetv to Main so that we can see a bit clearer in the various dependencies.